schemas/schema.gql

```diff
@@ -0,0 +1,14 @@
+type communities {
+  name: String!
+  address: String!
+  cid: String!
+  members: {
+    username: String!
+    address: String!
+    ens: String!
+  }
+}
```

Comment:
Couple thoughts here:

* Syntax is invalid, erroring when uploading to Fauna
* Need some documentation on how to bootstrap/setup Fauna DB from this schema
* Type should probably be singular and proper case
* Members should probably be it's own type
* Any indexes in Fauna should include some FQL either here or in the readme so a new dev can quickly setup the DB